libsql-server/src/namespace/meta_store.rs

```diff
@@ -128,6 +132,17 @@ impl MetaStoreInner {
             ",
             (),
         )?;
+        conn.execute(
+            "CREATE TABLE IF NOT EXISTS shared_schema_links (
+                shared_schema_name TEXT NOT NULL,
+                db_name TEXT NOT NULL,
```

Comment:
I was thinking about naming it namespace but since both shared schema and the other field are namespaces, I found it unclear and misleading. Naming the fields schema + db was meant to be a bit more informative but that's ok. Changed to namespace.